ms-deal/deal/src/main/java/com/enfint/deal/feign/FeignServiceUntil.java

```diff
@@ -0,0 +1,15 @@
+package com.enfint.deal.feign;
+
+import com.enfint.deal.dto.LoanApplicationRequestDTO;
+import com.enfint.deal.dto.LoanOfferDTO;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@FeignClient(value="clientData",url="http://localhost:8080/conveyer")
```

Comment:
It is better to get thing like that from application.properties/yaml

client.url.conveyor = http://localhost:8080/conveyer //application.properties

@FeignClient(value="clientData",url=conveyorUrl)